frontend/test/metabase/scenarios/dashboard/actions-on-dashboards.cy.spec.js

```diff
@@ -286,3 +530,18 @@ function createDashboardWithActionButton({
 
   saveDashboard();
 }
+
+const changeValue = ({ fieldName, fieldType, oldValue, newValue }) => {
+  cy.findByPlaceholderText(fieldName)
+    .should("have.attr", "type", fieldType)
+    .should("have.value", oldValue)
+    .clear()
+    .type(newValue);
+};
+
+const clickHelper = buttonName => {
+  // this is dirty, but it seems to be the only reliable solution to detached elements before cypress v12
+  // https://github.com/cypress-io/cypress/issues/7306
+  cy.wait(100);
+  cy.button(buttonName).click();
```

Comment:
sadly, I really don't think there's a way around this until we upgrade cypress. This will flake without the .wait(), and there's no other event that we can wait on here while the dashboard saves. We already wait on the relevant API requests in `saveDashboard()`

Reply:
But why does it get detached in the first place? @iethree

Reply:
we have lots of re-renders in dashboards 😢  I've been doing some thinking about dashboard refactoring that could improve this, and more importantly, dashboard performance as a whole 🤔